raiden/ui/web/src/app/components/channel-table/channel-table.component.spec.ts

```diff
@@ -1,25 +1,188 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClient, HttpHandler } from '@angular/common/http';
+import { async, ComponentFixture, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import {
+    ButtonModule,
+    DataTableModule,
+    DialogModule,
+    DropdownModule,
+    Menu,
+    MenuItem,
+    MenuModule,
+    TabViewModule
+} from 'primeng/primeng';
+import { of } from 'rxjs/internal/observable/of';
+import { Channel } from '../../models/channel';
+import { UserToken } from '../../models/usertoken';
+import { EllipsisPipe } from '../../pipes/ellipsis.pipe';
+import { KeysPipe } from '../../pipes/keys.pipe';
+import { SubsetPipe } from '../../pipes/subset.pipe';
+import { TokenPipe } from '../../pipes/token.pipe';
+import { RaidenConfig } from '../../services/raiden.config';
+import { RaidenService } from '../../services/raiden.service';
+import { SharedService } from '../../services/shared.service';
+import { EventListComponent } from '../event-list/event-list.component';
+import { OpenDialogComponent } from '../open-dialog/open-dialog.component';
 
 import { ChannelTableComponent } from './channel-table.component';
+import Spy = jasmine.Spy;
+
+export class MockConfig extends RaidenConfig {
+    public web3: any = {
```

Comment:
Nice you're implementing the tests. After you finish it, we may look into requiring it to be run together with the python tests. Just a comment =)
After, you may also look into [ganache-cli](https://github.com/trufflesuite/ganache-cli) from truffle suite, which actually runs a full tester node.

Reply:
I was thinking exactly the same thing.

Thanks for pointing ganache-cli I will check it out.